### DISCUSSION TITLE
Ability to unlink a project from the local environment

### DISCUSSION THREAD

**Discussion by rcwestlake** [supabase/supabase#17991] [Feature Requests]
We have the ability to run `supabase link --project-ref xyz` but we don't have the ability to unlink. I am using the Github Actions deploy strategy but needed to reset migrations that I messed up. I linked to my project but now I want to unlink as I don't want to accidentally do something to the remote project.

**Comment by AdditionAddict** [comment]
Yes, please can we unlink. I hate having my production linked unless I explicitly set this!

**Comment by eleijonmarck** [comment]
Added functionality of `unlink` to this PR. hopefully it gets merged
https://github.com/supabase/cli/pull/1849

**Comment by eleijonmarck** [comment]
@sweatybridge helped me get this merged so this could be used with the `unlink` command

**Comment by AdditionAddict** [comment]
@eleijonmarck tysm
